Pokemon_class.py

```diff
@@ -0,0 +1,36 @@
+class Pokemon:
+    """This is a class designed to hold relevant information for Pokemon mentioned on r/pokemon.
+
+    :param name: name of pokemon being assigned to the object
+    :type name: str
+    """
+
+    def __init__(self, name: str):
+        """constructor method"""
+        self.name = name
+        self.mentions = 1
```

Comment:
init this to 0, and then increment the mention when it's actually mentioned.  Think about this in the general case - there may be tons of times you'll instantiate a pokemon object for other purposes (testing, setting attributes, maybe getting things ready for later), so don't assume it's created just for the purposes of a mention.  

Also, see the code later for why - doing it starting with 1 is actually harder.